csp-mixin/signals/gcpce.libsonnet

```diff
@@ -0,0 +1,129 @@
+local commonlib = import 'common-lib/common/main.libsonnet';
+function(this)
+  {
+    local s = self,
+    filteringSelector: 'job!=""',
+    groupLabels: ['job', 'project_id'],
+    instanceLabels: ['instance_name'],
+    aggLevel: 'instance',
+    discoveryMetric: {
+      stackdriver: 'stackdriver_gce_instance_compute_googleapis_com_instance_cpu_utilization',
+    },
+    signals: {
+      // instancesText: {
+      //   name: "Instances",
+      //   type: 'text',
+      //   content: 'Use this section to look at one instance at a time by picking a value in the *Instance* picker at the top of the dashboard.'
+      //   // To Do: check if this works???
+      // },
+      cpuUtilization: {
+        name: 'CPU Utilization',
+        description: 'Fractional utilization of allocated CPU on this instance.',
+        type: 'raw',
+        unit: 'percentunit',
+        sources: {
+          stackdriver: {
+            expr: 'stackdriver_gce_instance_compute_googleapis_com_instance_cpu_utilization{%(queriesSelector)s}',
+            legendCustomTemplate: '{{instance_name}}',
+          },
+        },
+      },
+
+      networkReceived: {
+        name: 'Network received',
+        description: 'Count of bytes received from the network. Sampled every 60 seconds.',
+        type: 'counter',
+        unit: 'bytes',
+        aggFunction: 'avg',
+        sources: {
+          stackdriver: {
+            expr: 'stackdriver_gce_instance_compute_googleapis_com_instance_network_received_bytes_count{%(queriesSelector)s}',
+            legendCustomTemplate: '{{instance_name}}',
+          },
+        },
+      },
+      cpuUsageTime: {
+        name: 'Cpu usage time',
+        description: 'CPU usage, in seconds. For Container-Optimized OS, or Ubuntu running GKE.',
+        type: 'counter',
+        unit: 'seconds',
+        aggFunction: 'avg',
+        sources: {
+          stackdriver: {
+            expr: 'stackdriver_gce_instance_compute_googleapis_com_instance_cpu_usage_time{instance_name=~"$instance"}',
```

Comment:
Why not use all the filters here with `%(queriesSelector)s`? Now it says no data